SingleSource/UnitTests/Atomic/big_test.reference_output

```diff
@@ -0,0 +1,6 @@
+10 threads; 1000000 iterations each; total of 10000000
+Testing big
+CMPXCHG: atomic: 10000000 10000000 10000000 10000000 10000000 10000000 10000000 10000000 10000000 10000000 
+      nonatomic: 9970397 9970397 9970397 9970397 9972269 9972269 9972269 9972269 9941180 9941180 
```

Comment:
I'm not sure there is much value in including the non-atomic output in the reference (it could even match the atomic output in some cases, e.g. on a single CPU emulator). All we really care about in this test is that the atomic output is correct, right?

Would printing the non-atomic output to stderr instead of stdout keep the debugging feature while allowing us to drop the FP_TOLERANCE hack?

Reply:
Also thinking about this more, doesn't the FP_TOLERANCE comments also imply that the test will pass even if the values printed are wrong? So it would always require manual inspection?

Reply:
It's a good idea to move them to stderr. I'll do that.

The tests themselves check for correctness (`if (aint < expected) printf('fail')`), so the matcher will fail too.

Reply:
What is "traditional_output" in this context? https://github.com/llvm/llvm-test-suite/blob/main/litsupport/modules/timeit.py#L30

I'm having trouble separating stdout from stderr, since this timeit lit module doesn't support it & redirecting to test.out simultaneously. I think it would hamper test readability if I wrote my own debug logging, so I suppose we could drop all numerical logging. Is anyone against this?

Reply:
I've removed test result/numerical logging.